internal/pkg/callbacks/rolling_upgrade.go

```diff
@@ -67,6 +68,26 @@ func GetDeploymentItems(clients kube.Clients, namespace string) []runtime.Object
 	return items
 }
 
+// GetCronJobs returns the jobs in given namespace
+func GetCronJobs(clients kube.Clients, namespace string) []runtime.Object {
```

Comment:
Rename this to GetCronJobItems to stay consistent with the naming convention used in this file.